front/app/containers/IdeasShow/components/Container.tsx

```diff
@@ -68,7 +68,7 @@ interface Props {
   handleContainerRef: (element: HTMLElement | null) => void;
 }
 
-const Container = ({
+const Container2 = ({
```

Comment:
This shouldn't be a separate component; it doesn't add anything. In fact, it makes it harder to understand IdeasShow. So I'm not even trying to come up with a good name. This component needs to be removed